### PR TITLE
introduced proptypes

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/shared/components/Beer/Beer.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import './styles.scss';
+import PropTypes from 'prop-types';
 const Beer = ({ food, brewed, tagline, ebc, name, abv, description, img, ibu }) => {
```
Comment (mi-chal) [inline src/shared/components/Beer/Beer.jsx:3]: There should be a break between imports and component

File: src/shared/components/RandomBeer/RandomBeer.jsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { saveItemInLocalStorage, getItemFromLocalStorage } from '../../../services/localStorage';
 import Beer from '../Beer/Beer';
 import FavouriteButton from '../FavouriteButton/FavouriteButton';
+import PropTypes from 'prop-types';
 import './styles.scss';
 
 const RandomBeer = ({ value }) => {
```
Comment (mi-chal) [inline src/shared/components/RandomBeer/RandomBeer.jsx]: What is value? This name doesn't tell much

File: src/shared/components/Beer/Beer.jsx (hunk 1 above)
Comment (dwantula) [reply src/shared/components/Beer/Beer.jsx:3]: Done!

File: src/shared/components/RandomBeer/RandomBeer.jsx (hunk 2 above)
Comment (dwantula) [reply src/shared/components/RandomBeer/RandomBeer.jsx]: Fixed!

**Pull request merged**